docs/source/releasing.rst

```diff
@@ -41,11 +41,7 @@ Things to do for releasing:
 
 * tag with vVERSION, push tag on ``python-trio/trio`` (not on your personal repository)
 
-* push to PyPI::
-
-    git clean -xdf   # maybe run 'git clean -xdn' first to see what it will delete
-    python3 -m build
-    twine upload dist/*
+* approve the release workflow
```

Comment:
Nit: it's not the entire workflow but the publish job of the release workflow that will be paused until approved.